hardware/PhilipsHue/PhilipsHue.cpp

```diff
@@ -936,14 +943,32 @@ void CPhilipsHue::ReportTemperature(const int NodeID, const float temperature, c
 	tsen.TEMP.id1 = (sID & 0xFF00) >> 8;
 	tsen.TEMP.id2 = sID & 0xFF;
 	tsen.TEMP.tempsign = (temperature >= 0) ? 0 : 1;
-	int at10 = round(std::abs(temperature*10.0f));
+
+	int at10 = round(abs(temperature*10.0f));
 	tsen.TEMP.temperatureh = (BYTE)(at10 / 256);
 	at10 -= (tsen.TEMP.temperatureh * 256);
 	tsen.TEMP.temperaturel = (BYTE)(at10);
 
 	sDecodeRXMessage(this, (const unsigned char *)&tsen.TEMP, Name.c_str(), BatteryLevel);
 }
 
+
+void CPhilipsHue::ReportLux(const int NodeID, const float lux, const string &Name, uint8_t BatteryLevel)
```

Comment:
could you please remove this, and use the default way of reporting a (temperature) sensor ?
Just call

`void SendLuxSensor(const int NodeID, const int BatteryLevel, const float temperature, const std::string &defaultname);
`

These functions are specified in DomoticzHardware.cpp/h